pidcat.py

```diff
@@ -145,7 +146,10 @@ def allocate_color(tag):
 PID_KILL  = re.compile(r'^Killing (\d+):([a-zA-Z0-9._:]+)/[^:]+: (.*)$')
 PID_LEAVE = re.compile(r'^No longer want ([a-zA-Z0-9._:]+) \(pid (\d+)\): .*$')
 PID_DEATH = re.compile(r'^Process ([a-zA-Z0-9._:]+) \(pid (\d+)\) has died.?$')
-LOG_LINE  = re.compile(r'^([A-Z])/(.+?)\( *(\d+)\): (.*?)$')
+if args.add_timestamp:
```

Comment:
It doesn't make sense to have this logic sprinkled all over. Always capture the timestamp and just have the condition on whether or not it gets displayed.